src/BursarExportPlugin.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Button,
+  LoadingPane,
+  Pane,
+  PaneFooter,
+} from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+import { FormApi } from 'final-form';
+import React, { useCallback, useRef } from 'react';
+import formValuesToDto from './api/dto/to/formValuesToDto';
+import schedulingToDto from './api/dto/to/schedulingToDto';
+import useAutomaticSchedulerMutation from './api/mutators/useAutomaticSchedulerMutation';
+import useManualSchedulerMutation from './api/mutators/useManualSchedulerMutation';
+import ConfigurationForm, { FORM_ID } from './form/ConfigurationForm';
```

Comment:
it's not correct that you import form id in this component, make form id global or pass it as a prop

Reply:
Do you want it to be in the `src/BursarExportPlugin.tsx` file itself? If I do this I am getting a dependency cycle between this field and `src/form/ConfigurationForm.tsx`. And we can't really pass it as a prop as this is the main component.

Reply:
as I mentioned, you make it global (constants file on src level) or pass it as props (I didn't why it can't work - it shouldn't be a problem as for me)